testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -6162,6 +6162,18 @@ void checkRegexpExtract(SqlOperatorFixture f0, FunctionAlias functionAlias) {
     f.checkNull("log10(cast(null as real))");
   }
 
+  @Test void testLog2Func() {
+    final SqlOperatorFixture f = Fixtures.forOperators(true);
+    f.setFor(SqlStdOperatorTable.LOG2, VmName.EXPAND);
+    f.checkScalarApprox("log2(2)", "DOUBLE NOT NULL",
+        isWithin(1.0, 0.000001));
+    f.checkScalarApprox("log2(4)", "DOUBLE NOT NULL",
+        isWithin(2.0, 0.000001));
+    f.checkScalarApprox("log2(65536)", "DOUBLE NOT NULL",
```

Comment:
Can we add more tests:
with decimal arguments, negative arguments, maybe an expression (2 / 3)

Reply:
done, thank you